subworkflows/nf-core/vcf_extract_relate_somalier/tests/main.nf.test.snap

```diff
@@ -0,0 +1,938 @@
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test"
+                        },
+                        "NA12878.somalier:md5,39256adac134a8f304383250f8e062a4"
+                    ]
+                ],
+                "1": [
+                    [
+                        {
+                            "groupSize": 1,
+                            "groupTarget": {
+                                "id": "test"
+                            }
+                        },
+                        "test.html:md5,637558080f6e4a6b5f992523b11a7367"
+                    ]
+                ],
+                "2": [
+                    [
+                        {
+                            "groupSize": 1,
+                            "groupTarget": {
+                                "id": "test"
+                            }
+                        },
+                        "test.pairs.tsv:md5,4a0988fc3c0fe5cfd5dd205fe6755595"
+                    ]
+                ],
+                "3": [
+                    [
+                        {
+                            "groupSize": 1,
+                            "groupTarget": {
+                                "id": "test"
+                            }
+                        },
+                        "test.samples.tsv:md5,50fa1bf9b6844b29b57e3b7cc9dcecc5"
+                    ]
+                ],
+                "4": [
+                    "versions.yml:md5,1affbf55dcb6f7bc394ebc6ba8d32eaa",
+                    "versions.yml:md5,218f8c413e2214d0ec3c37ce942c6940",
+                    "versions.yml:md5,2be58dcf43db0e2b3be1a5f86b96cc45"
+                ],
+                "extract": [
+                    [
+                        {
+                            "id": "test"
+                        },
+                        "NA12878.somalier:md5,39256adac134a8f304383250f8e062a4"
+                    ]
+                ],
+                "html": [
+                    [
+                        {
+                            "groupSize": 1,
+                            "groupTarget": {
+                                "id": "test"
+                            }
+                        },
+                        "test.html:md5,637558080f6e4a6b5f992523b11a7367"
+                    ]
+                ],
+                "pairs_tsv": [
+                    [
+                        {
+                            "groupSize": 1,
+                            "groupTarget": {
+                                "id": "test"
+                            }
+                        },
+                        "test.pairs.tsv:md5,4a0988fc3c0fe5cfd5dd205fe6755595"
+                    ]
+                ],
+                "samples_tsv": [
+                    [
+                        {
+                            "groupSize": 1,
+                            "groupTarget": {
```

Comment:
Aah, a potential gotcha.
The `groupKey` leads to a structure that is different from normal. Probably worth undoing the groupKey with `.map { meta, tsv -> [meta.target, tsv] }`

Reply:
Make sense, I've added a check to remove it.